src/sass/app/_common/loading.scss

```diff
@@ -6,34 +6,12 @@
     white-space: nowrap;
 
     &__rect {
```

Comment:
Can you create a new sass file for this? (just duplicate the same file with changes you want to append). 

I still need to use this loader for the chart_loader, and want to avoid running into conflicts.

Reply:
it won't affect the generic loader as i change the initial loading file, it is indeed the same name 👍